docs/hub/hf-mcp-server.md

```diff
@@ -0,0 +1,61 @@
+# Hugging Face MCP Server
+
+The Hugging Face MCP (Model Context Protocol) Server connects your MCP‑compatible AI assistant (for example Codex, Cursor, VS Code extensions, Zed, or Claude Desktop) directly to the Hugging Face Hub. Once connected, your assistant can search and explore Hub resources and use community tools, all from within your editor or chat.
+
+## What you can do
+
+- Search and explore Hub resources: models, datasets, Spaces, and papers
+- Run community tools via MCP‑compatible Gradio apps hosted on [Spaces](https://hf.co/spaces)
+- Bring results back into your assistant with metadata, links, and context
+
+## Get started
+
+1. Open your MCP settings: visit https://huggingface.co/settings/mcp while logged in.
+
+2. Pick your client: select your MCP‑compatible client (for example Cursor, VS Code, Zed, Claude Desktop). The page shows client‑specific instructions and a ready‑to‑copy configuration snippet.
+
+3. Paste and restart: copy the snippet into your client’s MCP configuration, save, and restart/reload the client. You should see “Hugging Face” (or similar) listed as a connected MCP server in your client.
```

Comment:
Shouldn't we recommend to choose additional tools before restarting? Or maybe we can mention it later.